webservices/resources/aggregates.py

```diff
@@ -42,7 +42,7 @@ def build_query(self, committee_id=None, **kwargs):
 
 
 @doc(
-    tags=['schedules/schedule_a'],
+    tags=['receipts'],
```

Comment:
What does changing the tags here do; asked another way, how will this change manifest itself?